src/data-catalog/DataBrowserDetails.test.ts

```diff
@@ -0,0 +1,85 @@
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { h } from 'react-hyperscript-helpers';
+import { getDatasetReleasePoliciesDisplayInformation, useDataCatalog } from 'src/data-catalog/data-browser-utils';
+import { DataBrowserDetails, MainContent, MetadataDetailsComponent } from 'src/data-catalog/DataBrowserDetails';
+import { TEST_DATASET_ONE, TEST_DATASET_TWO, TEST_DATASETS } from 'src/data-catalog/test-datasets';
+import { asMockedFn, renderWithAppContexts as render } from 'src/testing/test-utils';
+
+const mockGoBack = jest.fn();
+
+type NavExports = typeof import('src/libs/nav');
+jest.mock(
+  'src/libs/nav',
+  (): NavExports => ({
+    ...jest.requireActual('src/libs/nav'),
+    history: { goBack: () => mockGoBack() },
+    getLink: jest.fn(),
+    useRoute: jest.fn(),
+  })
+);
+
+type DataBrowserUtilsExports = typeof import('src/data-catalog/data-browser-utils');
+jest.mock('src/data-catalog/data-browser-utils', (): DataBrowserUtilsExports => {
+  return {
+    ...jest.requireActual('src/data-catalog/data-browser-utils'),
+    useDataCatalog: jest.fn(),
```

Comment:
I think this could also use a separate `const` like `mockGoback` above. Or if the mock is the same, and we don't verify its calls, could you do the `mockReturnValue` here instead of in `beforeEach`?

Reply:
I don't think it needs a separate `const`, since we're not testing on it. I swapped to `beforeAll` though, rather than `beforeEach`. I think I prefer doing that to having the mockReturnValue outside of the describe.